com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/verifier/Verifier.java

```diff
@@ -0,0 +1,10 @@
+package com.sap.cloud.lm.sl.cf.process.util.verifier;
+
+import java.net.URL;
+import java.security.cert.X509Certificate;
+import java.util.List;
+
+public interface Verifier {
```

Comment:
This interface should also be unnecessary - especially when considering that one of the arguments of the `verify` method is called `jarURL`.

Reply:
The interface is removed